### ISSUE TITLE
Shadows not rendering properly on different devices

### ISSUE THREAD

**Issue by darko6977** [dotnet/maui#15565]
### Description

I ran into a problem when applying shadows to elements. The shadows do not get rendered properly on each device. This was tested on a virtual Pixel 5 (Android 13), Physical Xiaomi11T (Android 13) and Samsung J6(Android 10). It seems like the Radius property has no effect on the Xiaomi11T (MIUI 14.0.4 Global) and the shadows while the other two devices render the shadow properly. Below are screen shots of the problem.

Pixel 5:
![Pixel 5](https://github.com/dotnet/maui/assets/136151511/829ec161-2142-4c8c-bc0e-3f941ef02998)
Xiaomi 11T:
![Xiaomi 11T](https://github.com/dotnet/maui/assets/136151511/c2003e44-be54-47e4-9a83-edb0004df66c)

### Steps to Reproduce

Below is the code I used on a normal `ContentPage`:

```
<?xml version="1.0" encoding="utf-8" ?>
<ContentPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
             x:Class="Testiranje.TestPage"
             xmlns:vm="clr-namespace:Testiranje.ViewModels"
             Shell.NavBarIsVisible="False"
             Title="TestPage">
    <Grid>
        <ScrollView Margin="0,70,0,0">
            <VerticalStackLayout Spacing="5">
                <Grid HeightRequest="150" BackgroundColor="White" HorizontalOptions="Fill">
                    <RoundRectangle Margin="10" CornerRadius="15" Fill="white" Stroke="silver">
                        <RoundRectangle.Shadow>
                            <Shadow Brush="black" Offset="5,10" Radius="10" Opacity="0.5"/>
                        </RoundRectangle.Shadow>
                    </RoundRectangle>
                </Grid>
            </VerticalStackLayout>
        </ScrollView>
    </Grid>
</ContentPage>
```
TLDR code:
```
<Grid HeightRequest="150" BackgroundColor="White" HorizontalOptions="Fill">
    <RoundRectangle Margin="10" CornerRadius="15" Fill="white" Stroke="silver">
        <RoundRectangle.Shadow>
            <Shadow Brush="black" Offset="5,10" Radius="10" Opacity="0.5"/>
        </RoundRectangle.Shadow>
    </RoundRectangle>
</Grid>
```

### Link to public reproduction project repository

https://github.com/darko6977/NETMaui_Shadow_Bug/tree/5a9e53fc647b64a05380da1f422b7ffb52299d96

### Version with bug

7.0.49

### Last version that worked well

Unknown/Other

### Affected platforms

Android

### Affected platform versions

Android 13

### Did you find any workaround?

As of 10.06.2023. NO

**Comment by jsuarezruiz** [comment]
Seems the same as https://github.com/dotnet/maui/issues/10401

**Comment by darko6977** [comment]
> Seems the same as #10401

It seems it is the same. Should I close this one ?

**Comment by mattleibow** [comment]
Duplicate of https://github.com/dotnet/maui/issues/10401
